django/template/utils.py

```diff
@@ -106,3 +108,31 @@ def get_app_template_dirs(dirname):
             template_dirs.append(upath(template_dir))
     # Immutable return value because it will be cached and shared by callers.
     return tuple(template_dirs)
+
+
+def property_resolver(arg):
```

Comment:
I think this would be better as a private function in `defaultfilters.py` -- seems unlikely to be much of a generic utiltiy.